releasenotes/notes/primitives-v2-df871c0c6ac0b94a.yaml

```
@@ -0,0 +1,59 @@
+---
+features:
+  - |
+    Version 2 of the primitives was introduced. The emphasis of this new version is on performing
+    vectorized calls to the primitive run methods, so that sweeps over parameter value sets and 
+    observables can be efficiently specified and targetted against a single parametric circuit.
+
+    .. code: python
+
+        from qiskit.circuit import Parameter, QuantumCircuit
+        from qiskit.primitives import StateVectorSampler
```

```suggestion
        from qiskit.primitives import StatevectorEstimator
```
You use estimator in the sample, not sampler